addons/website_blog/models/website_blog.py

```diff
@@ -202,6 +202,13 @@ def _compute_teaser(self):
 
     def _set_teaser(self):
         for blog_post in self:
+            if not blog_post.with_context(lang='en_US').teaser_manual:
+                # By default, if no teaser is set in english, it will use the
+                # first 200 characters of the content. We don't want to break
+                # that when adding a manual teaser in a translation.
+                # That's how the ORM work: it will write the translation in the
+                # en_US default key too.
+                blog_post.update_field_translations('teaser_manual', {'en_US': ''})
```

Comment:
> - Both teaser (source and translated version) will show first 200 char
>   of the blog content
> - Enter translate mode (eg fr_BE) and add a manual teaser for french

I tested with your branch, I don't understand how this works. Seemed actually strange to allow that flow to happen: translating something whose source has not been set yet? How does that even work? I tested your PR thinking I would get this result:
- Go to blogs (all with auto-teasers)
- Enter translate mode
- Change one teaser (manual translated teaser)
- Go back to source language
- Change the manual teaser again ... basically just trying to "translate" it in the end (I am adding a teaser on a blog, I just started by the translated one and now I am adding the source one, weird).
=> Nice it works...
- Enter translate mode again... should not my manual translation be removed since the source changed? (delayed translation feature)

Apparently this is not what happens but I am not sure why :grimacing: 


> That's how the ORM work: it will write the translation in the
> en_US default key too.

I would change that comment I don't understand it. Is that the same as doing `blog_post.with_context(lang='en_US').teaser_manual = ''` or not? Or is this exactly what allows to not lose the translation changing the source term the first time?

Reply:
> I would change that comment I don't understand it. Is that the same as doing blog_post.with_context(lang='en_US').teaser_manual = '' or not? Or is this exactly what allows to not lose the translation changing the source term the first time?

It's how the ORM behaves **(for both text and html fields)**: when setting a translation value, if there is no source value, the source will also receive the translation value. And yes, whatever if you do it directly in a write or by calling `update_field_translations` (**only for TEXT fields**, I'll explain why below), that behavior will happen.

Eg, from this:
```
 id | teaser_manual 
----+---------------
  6 | 
(1 row)
```
Do this:
```
>>> self.env['blog.post'].browse(6).with_context(lang='fr_BE').teaser_manual = "fr teaser"
>>> self.env['blog.post'].browse(6).with_context(lang='fr_BE').teaser_manual
'fr teaser'
>>> self.env['blog.post'].browse(6).teaser_manual
'fr teaser'
```
Or this:
```
>>> self.env['blog.post'].browse(6).update_field_translations('teaser_manual', {'fr_BE': "fr teaser"})
>>> self.env['blog.post'].browse(6).with_context(lang='fr_BE').teaser_manual
'fr teaser'
>>> self.env['blog.post'].browse(6).teaser_manual
'fr teaser'
```
And you will end up with this:
```
 id |                teaser_manual                 
----+----------------------------------------------
  6 | {"en_US": "fr teaser", "fr_BE": "fr teaser"}
(1 row)
```

It's surely a nice feature in the backend and in Odoo in general, but in this particular case, it's not what we want, but since it's a very niche case, I am not surprised:
- This is about a **text** field (`teaser`)
- Which is actually a computed field
- Which is getting it's value from 2 other fields, the value either being extracted from an **HTML** field or being the value of a **TEXT** field
- Which is also translatable
- Which is interfaced in the frontend in a `t-field`

Finally, note that with HTML fields, since you don't translate it the same way (it's term by term translations, so possibly many values, not a single value like in text field), it does not behaves the same way.
For instance, you have to translate FROM a source term TO a translated value, so you can't add a translated value over an empty value.

About the delay translation, there is no such things for TEXT field, as there is no sync in the first place: when you modify the EN value, it does not modify the translation values.
While with html fields, any change on the source value will override translation values.
For instance:
```
>>> self.env['blog.post'].browse(6).content = "<p>rde</p><p>qsm</p>"
>>> self.env['blog.post'].browse(6).with_context(lang='fr_BE').content
Markup('<p>rde</p><p>qsm</p>'
>>> self.env['blog.post'].browse(6).with_context(lang='fr_BE').content = "<p>rde</p><p>qsm3</p>"
>>> self.env['blog.post'].browse(6).with_context(lang='fr_BE').content
Markup('<p>rde</p><p>qsm3</p>')
>>> self.env['blog.post'].browse(6).content = "<p>rde</p><p>qsm4</p>"
>>> self.env['blog.post'].browse(6).with_context(lang='fr_BE').content
Markup('<p>rde</p><p>qsm4</p>')
```

Hopefully this makes it more clear. All of this is stuff I actually learned investigating this bug, I was not familiar with any of this before.

Reply:
I replaced
```
                # That's how the ORM work: it will write the translation in the
                # en_US default key too.
```
by
```
                # That's how the ORM work: when setting a translation value, if
                # there is no source value, the source will also receive the
                # translation value
```
Should be more clear